### DISCUSSION TITLE
¿Dónde estarán compartiendo las grabaciones de las clases en vivo?

### DISCUSSION THREAD

**Discussion by Startupero** [Launch-X-Latam/MisionFrontEnd#22] [Q&A] [answered]
Para los que no podamos asistir a algunas de las clases en vivo ¿Dónde estarán compartiendo las grabaciones de las clases en vivo? Tomen en cuenta que no solo estudiantes están en el grupo, también profesionales de áreas afines, en mi caso trabajo y puede que en la semana solo pueda asistir a dos de las tres clases en vivo.

**Comment by JulioRamnses** [comment, marked as answer]
puedes encontrar las gabaciones en el general de teams, y podras ver a lado de donde dice "publicaciones", "archivos" y "grabaciones" 
![image](https://user-images.githubusercontent.com/114268025/192407450-96ee4d86-7cbe-49a9-bfcd-b5cf7d186016.png)

**Reply by KattGG** [reply]
Disculpen, ¿cómo entro al grupo de teams? no me mandaron ninguna liga ni código QR y al entrar con el usuario y contraseña que me proporcionaron, solo llené el formulario y ya no aparece nada.

**Reply by Startupero** [reply]
Tienes que llenar un formulario para que te inviten o agreguen al team LaunchX. https://forms.office.com/r/Ddr5fxJbNk

**Reply by KattGG** [reply]
De hecho entro y aparece este mensaje:

Your response has already been submitted
This form allows only one response per person.

**Reply by DaviiidSG** [reply]
Eso significa que ya lo respondiste, debes ingresar en Outlook con tu cuenta de launchX y ahí debería estar la invitación al teams

**Reply by JosafatJimenezB** [reply]
@KattGG checa tu correo de innovaccion, ahi te llegara el enlace de invitacion para unirte al teams

**Reply by KattGG** [reply]
Muchísimas gracias! No se porque lo estaba esperando en mi correo personal jeje
Mil gracias por su apoyo.

**Reply by Startupero** [reply]
Entra acá con los datos de acceso de LaunchX https://outlook.office.com/mail/
